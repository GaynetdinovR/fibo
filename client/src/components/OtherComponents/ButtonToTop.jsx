import styles from "../../styles/components/Other.module.sass";

const ButtonToTop = () => {
	return (
		<a href="#" className={styles.button_to_top}>
			<img src="./icons/top_arrow.png" alt="top_arrow" />
		</a>
	);
};

export default ButtonToTop;