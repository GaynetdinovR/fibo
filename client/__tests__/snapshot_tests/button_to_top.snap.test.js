import renderer from "react-test-renderer";
import { it, expect } from "@jest/globals";
import ButtonToTop from "../../src/components/OtherComponents/ButtonToTop.jsx";

it("renders correctly", () => {
	const tree = renderer.create(<ButtonToTop />).toJSON();
	expect(tree).toMatchSnapshot();
});
