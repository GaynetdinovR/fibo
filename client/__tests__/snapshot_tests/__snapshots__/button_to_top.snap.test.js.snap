// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="button_to_top"
  href="#"
>
  <img
    alt="top_arrow"
    src="./icons/top_arrow.png"
  />
</a>
`;
